libs/designsystem/src/lib/directives/size/size.directive.ts

```diff
@@ -2,8 +2,9 @@ import { Directive, Input, HostBinding } from '@angular/core';
 
 @Directive({
   // don't worry. I know what i am doing!
-  // tslint:disable-next-line:directive-selector
-  selector: 'button[size], kirby-icon[size], kirby-avatar[size], kirby-item[size]',
+  selector:
+    // tslint:disable-next-line: directive-selector
+    'button[size],kirby-icon[size], kirby-avatar[size], kirby-item[size],kirby-text-link[size]',
```

Comment:
Consider moving to HostBinding inside the component instead (see: `libs\designsystem\src\lib\components\progress-circle\progress-circle.component.ts` for an example):
```
@HostBinding('class')
  get _cssSize() {
    return this.size;
  }
```